README.md

```diff
@@ -59,6 +59,11 @@ impl BridgeTokenFactory {
     /// Expects ~35N attached to cover storage for BridgeToken.
     #[payable]
     pub fn deploy_bridge_token(address: String);
+
+    /// Checks if Bridge Token has been successfully deployed with `deploy_bridge_token`.
+    /// On success, returns the name of NEP21 contract associated with given address (<hex(evm_address)>.<current_id>).
+    /// Otherwise, returns "token do not exists" error.
+    pub fn get_bridge_token_account_id(&self, address: String) -> AccountId;
```

Comment:
Does this return Borsh- or JSON-encoded data?

Reply:
It seems all borsh-serialized methods has `#[result_serializer(borsh)]`. This method doesn't.
If we want to add info about borsh serialization, we should do it with all methods, (`finish_withdraw` for example). However, we want to keep README as light as possible to onboard developers quickly. Personally, I don't think we need to differentiate methods result serialization in README.